src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_NUMBER = 100;
+    private final Random random = new Random();
+    private ColorSupplier cs = new ColorSupplier();
```

Comment:
programmers like informative names
maybe we can find a better one for `cs`
also, `RANDOM_NUMBER` isn't so _random_, we can find a better name